packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -491,10 +465,12 @@ const CFDPersonalDetailsForm = ({
                                                                 name={field.name}
                                                                 list={account_opening_reason}
                                                                 value={values.account_opening_reason}
+                                                                disabled={is_account_opening_reason_disabled}
                                                                 onChange={handleChange}
                                                                 handleBlur={handleBlur}
                                                                 error={account_opening_reason_error}
                                                                 list_portal_id='modal_root'
+                                                                test_id='account_opening_reason'
```

Comment:
Nit: May be it can be called `dt_account_opening_reason` as called in the Test guide?